features/messages/impl/src/test/kotlin/io/element/android/features/messages/textcomposer/MessageComposerPresenterTest.kt

```diff
@@ -304,6 +331,14 @@ class MessageComposerPresenterTest {
             assertThat(messageSentState.richTextEditorState.messageHtml).isEqualTo("")
             assertThat(messageSentState.canSendMessage).isFalse()
             assertThat(fakeMatrixRoom.replyMessageParameter).isEqualTo(A_REPLY to A_REPLY)
+            assertThat(analyticsService.capturedEvents).containsExactly(
+                Composer(
+                    inThread = false,
+                    isEditing = false,
+                    isReply = true,
+                    messageType = Composer.MessageType.Text,
```

Comment:
Just a remark, when you reply with EXA to a message which is in thread, the message will be sent to the thread. This is handled by the SDK. Not sure that we want to reflect this in the analytics. I guess we will use `inThread = true` only when we will properly support threads.